.github/actions/print-config-summary/action.yml

```
@@ -0,0 +1,17 @@
+name: Print summary of important configuration
+description: Contains .NET, Node.js, and pnpm versions, among others
+
+runs:
+  using: "composite"
+  steps:
+    - name: Write configuration to summary
+      shell: pwsh
+      run: |
+        $toolVersions = @"
+          | Tool | Version |
+          | ---- | ------- |
+          | dotnet | $(dotnet --version) |
+          |  node  | $(node -v) |
+          |  pnpm  | $(pnpm -v) |
```

Single space around cell content and three hyphens are enough.

With alignment:
```suggestion
          | Tool | Version |
          | :--- | ---: |
          | dotnet | $(dotnet --version) |
          | node | $(node -v) |
          | pnpm | $(pnpm -v) |
```

---

I liked my version, but if you insist... I think either is fine.

But I like the improved alignment.

---

So, how about:

```suggestion
          | Tool | Version |
          | :--- | ------: |
          | dotnet | $(dotnet --version) |
          | node | $(node -v) |
          | pnpm | $(pnpm -v) |
```